spec/install/gemspecs_spec.rb

```diff
@@ -63,6 +63,31 @@
     expect(out).to include("Bundle complete!")
   end
 
+  it "reads gemspecs respecting their encoding" do
+    create_file("version.rb", <<-RUBY)
+      module Persistent💎
+        VERSION = "0.0.1"
+      end
+    RUBY
+
+    create_file("persistent-dmnd.gemspec", <<-G)
+      require_relative "version"
+
+      Gem::Specification.new do |gem|
+        gem.name = "persistent-dmnd"
+        gem.version = Persistent💎::VERSION
+        gem.author = "Ivo Anjo"
+        gem.summary = "Unscratchable stuff"
+      end
+    G
```

Comment:
Looking at how nice the `version.rb` file in this diff looks, we might want to change the `G` tag to `RUBY` everywhere :)